docs/getting-started-scp.md

```diff
@@ -0,0 +1,90 @@
+# Installing AWS Service Broker on SAP Cloud Platform
+This guide helps you integrate AWS services into the Cloud Foundry environment on SAP Cloud Platform by using the AWS Service Broker.
+
+For a high-level overview of how this works, take a look at the following image:
+
+![SCP AWS Service Broker integration](/docs/images/scp-aws-service-broker.png)
+
+## Pre-requisites
+
+* Account on [SAP Cloud Platform, Cloud Foundry environment](https://cloudplatform.sap.com/index.html).
+* Account on [AWS](https://aws.amazon.com/free/?awsf.Free%20Tier%20Types=*all&all-free-tier.sort-by=item.additionalFields.SortRank&all-free-tier.sort-order=asc#featured).
+* [Cloud Foundry CLI](https://help.sap.com/viewer/e275296cbb1e4d5886fa38a2a2c78c06/Cloud/en-US/4a8eb630c2734c01a25090c51d48102b.html).
+* [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git).
+
+## Configuration in your AWS Environment
+
+1. Sign in to your AWS account and in the AWS console, search for Cloudformation.
```

Comment:
CloudFormation has the `F` capitalized, this needs to be updated in several places in this pr